scrapers/vogov.yml

```
@@ -0,0 +1,24 @@
+name: vogov
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - vogov.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@property="og:title"]/@content
+      Date:
+         selector: //ul[@class='list-unstyled info-video-details']/li[1]/span[1]
+         postProcess:
+           - parseDate: Jan 02, 2006
```

A little safer to use and also single digit days don't have a `0`
```suggestion
      Date:
        selector: //div[@class="video-page-description"]//li[contains(.,"Release Date")]/span
        postProcess:
          - parseDate: Jan 2, 2006